docs/integrations/pytorch.rst

```diff
@@ -3,109 +3,370 @@
+    - :ref:`Catalyst <integrations-catalyst>`
+    - :ref:`Skorch <integrations-skorch>`
+
+    For pure PyTorch integration, read on.
+
+|colab-script-neptune|
+
+What will you get with this integration?
+----------------------------------------
+
+|pytorch-tour-loom|
+
+|Pytorch| is an open source machine learning framework commonly used for building deep neural network models.
+Neptune helps with keeping track of model training metadata.
+
+With Neptune + PyTorch integration you can:
+
+- log hyperparameters
+- see learning curves for losses and metrics during training
+- see hardware consumption during training
+- log stdout and stderr
+- log training code and .git information
+- log model weights
+- log torch tensors as images
+
+.. tip::
+    You can log many other experiment metadata like interactive charts, video, audio and more.
+    See a :ref:`full list <what-you-can-log>`.
+
+.. note::
+
+    This integration is tested with ``torch==1.7.0``, ``neptune-client==0.4.126``.
 
-.. image:: ../_static/images/integrations/pytorch_neptuneml.png
-   :target: ../_static/images/integrations/pytorch_neptuneml.png
-   :alt: Pytorch neptune.ai integration
+Where to start?
+---------------
+To get started with this integration, follow the :ref:`quickstart <pytorch-quickstart>` below.
+You can also skip the basics and take a look at how to log model weights and prediction images in the :ref:`advanced options <pytorch-advanced-options>` section.
 
-Log PyTorch metrics
--------------------
-It is very easy to track PyTorch metrics in Neptune. The following example presents a simple integration that lets you track loss:
+If you want to try things out and focus only on the code you can either:
+
+|colab-script-neptune|
+
+.. _pytorch-quickstart:
+
+Quickstart
+----------
+This quickstart will show you how to:
+
+* Install the necessary neptune packages
+* Connect Neptune to your Optuna hyperparameter tuning code and create the first experiment
+* Log metrics, figures, and artifacts from your first Optuna sweep to Neptune, and
+* Explore them in the Neptune UI.
+
+.. _pytorch-before-you-start-basic:
+
+Before you start
+^^^^^^^^^^^^^^^^
+You have ``Python 3.x`` and following libraries installed:
+
+* ``neptune-client``. See :ref:`neptune-client installation guide <installation-neptune-client>`.
+* ``torch``. See |pytorch-install|.
+
+You also need minimal familiarity with torch. Have a look at this |pytorch-guide| to get started.
+
+.. code-block:: bash
+
+   pip install --quiet torch neptune-client
+
+Step 1: Initialize Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
+
+.. code-block:: python3
+
+    import neptune
+
+    neptune.init(api_token='ANONYMOUS', project_qualified_name='shared/pytorch-integration')
+
+.. tip::
+
+    You can also use your personal API token. Read more about how to :ref:`securely set the Neptune API token <how-to-setup-api-token>`.
+
+Step 2: Create an Experiment and log parameters
```

Comment:
you did not log params in this step.

Perhaps you can cover init and create_exp in the step 1.